### DISCUSSION TITLE
LOUPEDECK 5.10.12043 problem

### DISCUSSION THREAD

**Discussion by PathTracer75** [calibx/msfsdeck#86] [General]
Hi, I hope you're fine first of all!

I don't know if you still working on this project but I have some issues with your plugin and the new LOUPEDECK driver 5.1 which came out some days ago. Basically each time I start MSFS the LOUPEDECK LIVE device starts successfully the profile but after a few seconds it is completely blocked! I need to restart the PC to restore it.

FYI, I use Windows 11 Latest Insiders Beta + FS2020 SU09.

Thanks in advance and have a nice weekend!

**Comment by calibx** [comment]
Hi,

Yes ! I am working on a new release since last month to comply with the new 64 Bit Loupedeck driver.
It takes some time because I rewrited all the event model to fix the refreshing problem and to remove the needs of FSUIPC
I hope to release a new version tomorrow with all the fixes.

**Comment by PathTracer75** [comment]
Hi again and thanks for getting back.

Good news then and I am really glad for the new version to come.
At this point I'll uninstall the previous one and will wait for the new.

Take care!

**Reply by calibx** [reply]
1.0.0 is online !

**Reply by PathTracer75** [reply]
Great news, very kind writing to me here!
I'll download it and let you know right away.

Thanks for the work you're doing as always!

**Comment by PathTracer75** [comment]
It works perfectly and better than before! Great job buddy!
